### DISCUSSION TITLE
24

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#541] [Comments]
http://jojoldu.tistory.com/24

**Comment by jojoldu** [comment]
작성자: 조우진

아 정말 가고 싶었지만 참석 못했던 모임인데 정성스런 후기 너무나 감사합니다 :)

**Comment by jojoldu** [comment]
작성자: 창천향로

ㅎㅎ 조금이나 아쉬움 달래셨다면 다행입니다^^ 방문해주셔서 감사합니다!!

**Comment by jojoldu** [comment]
작성자: 윤석준

이런 행사가 있는지도 몰라서 못갔는데, 그나마 후기로라도 볼 수 있어서 다행이란 생각이 드네요. 너무나 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

행사를 몰랐던 분들이 생각보다 많으시더라구요! 다행이라고 말씀해주셔서 정말 감사합니다^^

**Comment by jojoldu** [comment]
작성자: 장동민

감사합니다. 행사가 열리는지도 모르고 있었는데 덕분에 많은 도움이 되었습니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

넵! ㅎ 저도 다른분들의 후기덕분에 아쉬움 달랜적이 많아 저도 도움이 되고자 했는데 도움이 되셔서 정말 다행입니다! 방문해주셔서 감사합니다^^

**Comment by jojoldu** [comment]
작성자: camphora

다른 모임이 있어 못갔는데 후기 남겨주셔서 감사합니다. 못간것의 아쉬움도 남네요.

**Comment by jojoldu** [comment]
작성자: 창천향로

ㅎㅎ 저도 직접 라이브를 볼수있어서 다행이라고 생각했습니다! 멘토님들의 발표자료는 웹에 공개되는대로 바로바로 블로그에 기재하겠습니다^^

**Comment by jojoldu** [comment]
작성자: 임도형

잘 정리되어네요. 좋은 내용 아쉬웠는데, 유익했습니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

유익하셨다니 다행입니다^^ 방문해주셔서 감사합니다!

**Comment by jojoldu** [comment]
작성자: Outsider

정말 잘 정리해주셔 저도 다시 읽으면서 재밌었네요.. 재밌게 들으셨다니 다행입니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

으악 ㅋㅋㅋㅋㅋ 아웃사이더님 ㅠㅠ 방문해주셔서 감사합니다!!! 미흡한 글이지만 재밌게 읽어주셔서 정말 감사합니다!!

**Comment by jojoldu** [comment]
작성자: x멘토

잘 정리해주셨네요. 감사합니다. 재미있을까 준비하면서 운영진과 멘토들이 다양한 의견도 나누고 고민했는데 이렇게 좋은 피드백을 주셔서 큰 위안과 힘이 되네요. 
다음에도 더 재미있고 유익한 세미나 기획해보겠습니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

헉 x멘토님!! ㅋㅋㅋㅋ 좋은 피드백이라니 ㅠㅠ 극찬 감사드립니다!! 저역시도 다음에 멘토님의 세미나 참석하게되면 더더 좋은 후기를 남기도록 노력하겠습니다.
매번 너무나 감사드립니다!!

**Comment by jojoldu** [comment]
작성자: seungdols

구글 I/O를 가느라 가지 못했는데, 상세하고 정리 잘 된 후기 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

ㅎㅎ 저도 구글 IO에 갈까 KSUG 세미나에 갈까 고민했었는데! 도움이 되셨다니 다행입니다!

**Comment by jojoldu** [comment]
작성자: 김군

와 정말 잘 들었습니다. 저도 구글 IO행사가느라 못갔는데, 아까울 정도로 좋은 행사였다는게 느껴집니다!! ^^

**Comment by jojoldu** [comment]
작성자: 창천향로

넵! ㅎㅎ 정말 좋은행사였습니다! 다음에는 김군님도 참가하시면 좋으실것 같습니다!! 방문해주셔서 감사합니다!

**Comment by jojoldu** [comment]
작성자: 황양

와!! 가고싶었지만 시간이 안되서 못갔었는데... +_+ 이런 귀한 후기를 올려주시다니!!! 피와 살이 되는 후기 감사합니다!!
이 글을 읽고 또 도전받습니다!! +_+ 열공해야겠어요!~

**Comment by jojoldu** [comment]
작성자: 창천향로

황양님!! 방문해주셔서 감사합니다^^ 열정에 조금이나마 도움이 되셨다니 제가더 감사합니다^^ 또 방문 부탁드려요!! ㅎㅎ

**Comment by jojoldu** [comment]
작성자: 황양

와!! 가고싶었지만 시간이 안되서 못갔었는데... +_+ 이런 귀한 후기를 올려주시다니!!! 피와 살이 되는 후기 감사합니다!!
이 글을 읽고 또 도전받습니다!! +_+ 열공해야겠어요!~

**Comment by jojoldu** [comment]
작성자: asbubam

안녕하세요. 이승우입니다. 저도 참여해서 무척 즐거웠던 행사였는데, 후기올려주시고, 링크도 걸어주셔서 제 이력서 뷰가 많이 올랐습니다. :)
글을 읽다보니, 행사내용도 더 기억에 오래남고 좋은 것 같아요. 좋은 글 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

옷!!! 승우님 안녕하세요!! 이전부터 페이스북에서 계속 승우님의 글을 눈팅하고 있었는데 이렇게 뵙게되서 정말 영광입니다^^
조금이나마 도움이 되셨다니 정말 다행입니다^^ 다음 세미나때는 승우님의 발표를 play node에 이어 또 볼수있길 고대하고 있겠습니다!
방문해주셔서 감사합니다^^

**Comment by jojoldu** [comment]
작성자: 사이먼

소중한 경험 공유해주셔서 감사드립니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

넵! 사이먼님도 소중한 방문 해주셔서 감사합니다!!

**Comment by jojoldu** [comment]
작성자: 1466974722

잘보고가요~

**Comment by jojoldu** [comment]
작성자: 창천향로

방문해주셔서 감사합니다^^

**Comment by jojoldu** [comment]
작성자: HSJ

현장에 참여할수 없었지만.. 이 글을보고 새로움 다짐이 생기네요^^
잘봤습니다.

**Comment by jojoldu** [comment]
작성자: 창천향로

잘보셨다니 제가 도리어 감사드립니다^^
다음에도 좋은 후기 남기도록 하겠습니다!

**Comment by jojoldu** [comment]
작성자: 안여름

정성스러운 후기 작성해 나눠주셔서 감사합니다. 생생한 후기 덕분에 마치 세미나에 다녀온 듯 하네요:) 열정과 동기부여 잔뜩 받고 갑니다!! :)

**Comment by jojoldu** [comment]
작성자: 창천향로

여름님! 방문해주셔서 감사합니다! 당시에는 내용을 타이핑하느라 조금 고생스러웠는데 여름님 댓글을 보니 하길 잘했다는 생각이 드네요!
방문해주셔서 정말 감사드립니다^^

**Comment by jojoldu** [comment]
작성자: jinmu

세미나에 참석 못한 아쉬움을 달래기에 충분이 좋은 후기 였습니다. 많이 배우고 갑니다. 감사합니다!!

**Comment by jojoldu** [comment]
작성자: 백승우

우연히 들렀는데 정말 잘보고 갑니다..저를 반성하게 되네요

**Comment by cheese10yun** [comment]
좋은 정보 감사합니다.

**Comment by jojoldu** [comment]
@cheese10yun 잘봐주셔서 감사합니다^^

**Comment by aliwo** [comment]
와... 이 통찰력을 얻기 위해 다들 얼마만큼의 시간과 노력을 쏟아부으셨을지... 으마으마합니다!

**Comment by gf0308** [comment]
정말 이런 좋은 글 올려주셔서 감사합니다
한줄한줄 메모한것이 정말 많았을정도로
너무 유익하고 필요한 귀한 내용이었습니다
선한 영향력에 감사드립니다
